x2b_default_config.toml

```diff
@@ -0,0 +1,12 @@
+[slurm-args]
+time = "04:00:00"
+mem = 16000
+nodes = 1
+cpus-per-task = 2
+job-name = "xnat2bids"
```

Comment:
```suggestion
job-name = "xnat2bids"
output = "/gpfs/scratch/%u/logs/xnat2bids%J.log"
```

Reply:
In PR #11, I'm not specifying an `output` parameter in `slurm-args`, so that I can add the session # to the logging statements. 

`       # Define output for logs
        if not ('output' in arg_dict['slurm-args']):
            output = f"/users/{user}/logs/%x-{session}-%J.txt"
            arg = f"--output {output}"
            slurm_param_list.append(arg)`
            
But I like the idea of using %u, and letting slurm define user here.  Should logs go to `scratch/user/logs` vs. `/users/user/logs` ?